projects/plugins/jetpack/extensions/blocks/story/player/components/header.js

```diff
@@ -18,7 +18,7 @@ export default function Header( { fullscreen, onExitFullscreen, siteIconUrl, sto
 	return (
 		<div className="wp-story-meta">
 			<div className="wp-story-icon">
-				<img alt="Site icon" src={ siteIconUrl } />
+				<img alt="Site icon" src={ siteIconUrl } width="40" height="40" />
```

Comment:
This make me notice that Site Icon should probably be translated.

Reply:
Right! Done in 4415f90.